src/web/mjs/engine/Settings.mjs

```diff
@@ -253,6 +253,20 @@ export default class Settings extends EventTarget {
             ],
             value: 'none'
         };
+
+        this.NovelDarkmode = {
+            label: 'Light/Dark mode for Novels',
+            description: [
+                'choose between light and dark mode for novels',
+                'this may not work for all connectors that support novels'
+            ].join('\n'),
+            input: types.select,
```

Comment:
Maybe a checkbox would be more appropriate

Reply:
should 
![](https://i.imgur.com/RPpJyIo.png)
also be a check box?

other background/text colours are viable 
(im aware that it should be different if im gonna make that argument)

Reply:
No, this is a selection list, because there can be 1...n opyions, but i think for light/dark mode there will only just be the two of them.